__tests__/app.test.js

```diff
@@ -126,5 +126,48 @@ describe('GET /api/articles/:article_id', () => {
         .then(({ body: { msg } }) => {
             expect(msg).toBe('Invalid article_id');
         });
+  });
 });
-});
\ No newline at end of file
+
+describe('GET /api/articles/:article_id/comments', () => {
+  test('200: Responds - each comment should have the correct structure', () => {
+    const article_id = 1; 
+    return request(app)
+      .get(`/api/articles/${article_id}/comments`)
```

Comment:
You need to check that the array is the length you expect before using the forEach. If you try to iterate through an empty array, it won't happen and all your assertions will be skipped over.